OpenRA.Game/Graphics/Palette.cs

```diff
@@ -144,6 +144,16 @@ public ImmutablePalette(IEnumerable<uint> sourceColors)
 			foreach (var sourceColor in sourceColors)
 				colors[i++] = sourceColor;
 		}
+
+		public ImmutablePalette(IEnumerable<uint> sourceColors, int[] remapShadow)
```

Comment:
Can call `:this(sourceColors)` here to save some code repetition.

Reply:
Good idea. Thanks.